src/client/common/installer/productInstaller.ts

```diff
@@ -145,11 +142,14 @@ export class DataScienceInstaller extends BaseInstaller {
             action: 'displayed',
             moduleName: productName
         });
-        const item = await this.appShell.showErrorMessage(
-            localize.DataScience.libraryNotInstalled().format(productName),
-            'Yes',
-            'No'
-        );
+        const item = this.serviceContainer.get<boolean>(IsCodeSpace)
+            ? 'Yes'
+            : await this.appShell.showErrorMessage(
+                  localize.DataScience.libraryNotInstalled().format(productName),
+                  'Yes',
```

Comment:
Shouldn't 'Yes'/'No' be localized?

Reply:
Yes, I just kep the existing code as is.
Will fix.